templates/site/elaomojis.html.twig

```diff
@@ -26,20 +26,22 @@
         </div>
 
         {% for category in config.categories %}
```

Comment:
💡 **suggestion:** Plutôt que de faire un if dans la boucle, tu peux effectuer un [filtre](https://twig.symfony.com/doc/3.x/filters/filter.html) en amont directement dans ce for: 

```suggestion
        {% for category in config.categories|filter(c => not c.hidden ?? false) %}
```

Reply:
<img width="393" alt="Capture d’écran 2024-08-06 à 10 24 38" src="https://github.com/user-attachments/assets/9b731634-28b9-4d10-93d3-6f9ff1f2828e">